public/app/plugins/datasource/cloud-monitoring/datasource.ts

```diff
@@ -179,7 +179,18 @@ export default class CloudMonitoringDatasource extends DataSourceWithBackend<
           const dataQueryResponse = toDataQueryResponse({
             data: data,
           });
-          return dataQueryResponse?.data[0]?.meta?.custom?.labels ?? {};
+          return _map(dataQueryResponse?.data, 'meta.custom.labels')
+            .filter((p) => !!p)
+            .reduce((acc, labels) => {
+              for (let key in labels) {
+                if (acc[key]) {
+                  acc[key].push(...labels[key]);
```

Comment:
You just want to push the key, or am I missing something? 
`acc[key].push(labels[key])`

Reply:
It is wrong code.
And I notice that response format is not appropriate.

The value type of label map should be string, not string list.
I fixed the response format in https://github.com/grafana/grafana/pull/44365/commits/fc7bb7c5dc48ec964b195e4ec7241d423ed96030 .

And, fixed frontend in https://github.com/grafana/grafana/pull/44365/commits/1b4ed422719df6e70d3d0e9746fafd8ce08911af .